src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -50,38 +81,40 @@ async function initializeConnection(circuitHandlers: CircuitHandler[]): Promise<
 
   connection.on('JS.BeginInvokeJS', DotNet.jsCallDispatcher.beginInvokeJSFromDotNet);
   connection.on('JS.RenderBatch', (browserRendererId: number, renderId: number, batchData: Uint8Array) => {
```

Comment:
The `renderId` here is now `batchId`, isn't it?

Reply:
yes. browserRenderId, renderId was making my head explode.